From 8a331dda785427fb614f36ee34fba1c1be4be18d Mon Sep 17 00:00:00 2001
From: Manigandan Gopalakrishnan <Manigandan_Gopalakrishnan@comcast.com>
Date: Mon, 20 Jul 2026 19:38:30 +0530
Subject: [PATCH] RDKBDEV-3469 RDKBACCL-1961: Error flooding in
 Consolelog.txt.0 during log upload

Reason for change: Getting current wan if_name from sysevent
Test procedure: Not observing the error in Consolelog.txt.0
Risks: Low
Priority: P0

Signed-off-by: Manigandan Gopalakrishnan <Manigandan_Gopalakrishnan@comcast.com>
---
 arch/intel_usg/boards/arm_shared/scripts/network_response.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/intel_usg/boards/arm_shared/scripts/network_response.sh b/arch/intel_usg/boards/arm_shared/scripts/network_response.sh
index 78e1d18..4b98c8c 100644
--- a/arch/intel_usg/boards/arm_shared/scripts/network_response.sh
+++ b/arch/intel_usg/boards/arm_shared/scripts/network_response.sh
@@ -26,7 +26,11 @@
 source /lib/rdk/t2Shared_api.sh
 
 TRIGGER_STATE=$1
+WAN_INTERFACE="$(sysevent get current_wan_ifname 2>/dev/null)"
+if [ -z "$WAN_INTERFACE" ]
+then
 WAN_INTERFACE="erouter0"
+fi
 REVERT_FLAG="/nvram/reverted"
 EROUTER_IP=""
 EROUTER_IP6=""
-- 
2.43.0

